server/Controller/userController.js

```diff
@@ -0,0 +1,58 @@
+
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+import CryptData from '../Utilities/cryptData';
+import pool from '../Model/db/connectToDb';
+import queries from '../Model/queries';
+
+dotenv.config();
+const secret = process.env.SECRET;
+
+/**
+ * Class representing the user controller
+ * @description usercontroller
+ */
+class UserController {
+  /**
+   * @param {object} req -Request object
+   * @param {object} res - Response object
+   * @return {object} - Response object
+   */
+  static userSignUp(req, res) {
+    const { username, password, email } = req.body;
+    let encryptedPassword;
+    CryptData.encryptData(password).then((hash) => {
+      encryptedPassword = hash;
+      pool.query(queries.signup, [username, encryptedPassword, email],
+        (error, response) => {
+          if (error) {
+            res.status(409).jsend.fail({
+              code: 409,
+              message: 'User details allready exist. Signup was not successful',
```

Comment:
`allready` should be `already`

Reply:
The typo-error has been corrected and other response messages have been checked. A PR to implement this feedback is via his link: https://github.com/Daymorelah/orderMeal/pull/27